### PR TITLE
Fix: Do not summarize multiple replies in conversastion list subtitle

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

Fix the issue that multiple replies are summarized in the conversation list subtitle.

Updated string resources for ephemeral replies.

### CODE REVIEW

File: Wire-iOS/Resources/Base.lproj/Localizable.strings [hunk 1]
```diff
@@ -218,8 +218,8 @@
 "conversation.status.message.ephemeral.group" = "Someone sent a message";
 "conversation.status.message.ephemeral.mention" = "Mentioned you";
 "conversation.status.message.ephemeral.mention.group" = "Someone mentioned you";
-"conversation.status.message.ephemeral.reply" = "Replied you";
-"conversation.status.message.ephemeral.reply.group" = "Someone replied you";
+"conversation.status.message.ephemeral.reply" = "Replied your message";
```
Comment (typfel) [inline Wire-iOS/Resources/Base.lproj/Localizable.strings]: This reads wrong, shouldn't it be "Replied to your message"

Comment (typfel) [reply Wire-iOS/Resources/Base.lproj/Localizable.strings]: Also design updated the push notification copy from "Someone replied to your message" to "Someone replied to you". Maybe they want the similar copy changes here? Can you check with them?

**Comment by mikeger** [comment]
@billypchan why this one is marked as not to be merged?

**Comment by typfel** [comment]
It still needs the correct copy.

**Comment by billypchan** [comment]
@mikeger @typfel the string is corrected.

**Pull request merged**